ffi/ffi_serialization.hpp

```diff
@@ -0,0 +1,64 @@
+#ifndef __LIBFF_FFI_FFI_SERIALIZATION_HPP__
+#define __LIBFF_FFI_FFI_SERIALIZATION_HPP__
+
+#include <libff/algebra/curves/public_params.hpp>
+
+namespace libff
+{
+
+namespace ffi
+{
+
+// FFI buffers are expected to be big-endian formatted, and padded on
+// the left to be the same size as the in-memory bigint
+// representations.  Coefficients of field elements and, and
+// coordinates of curve points are placed consecutively in memory
```

Comment:
and affine coordinates (may be worth precising the coordinate form here)

Reply:
added note